content-src/experiments/tracking-protection.yaml

```diff
@@ -86,6 +86,7 @@ contributors:
     display_name: 'Rebecca Billings'
     title: 'Senior QA Engineer'
     avatar: /static/images/experiments/tracking-protection/avatars/rebecca-billings.jpg
+contributors_extra: 'This experiment is based on Firefox Tracking Protection technology built by Mozilla employees and contributors over the past several years. Learn more about <a href="https://mzl.la/2fYneDq" target="blank">Firefox Tracking Protection</a>.'
```

Comment:
No can do, it needs to be there, unfortunately.  I thought community L10Ns could navigate URLs, no